MANIFEST.in

```diff
@@ -1,31 +1,41 @@
-include MANIFEST.in
+# Reference: https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html
+
+# Include source files in SDist
 include CMakeLists.txt
+include *.bzl
+include *.bazel
+include WORKSPACE
+include requirements.txt
+include version.txt
+include .gitmodules
+graft android
+graft aten
+graft binaries
+graft c10
+graft caffe2
+graft cmake
+graft functorch
+graft third_party
```

Comment:
Ugh... this includes all the `.git` files in submodules recursively. That will massively inflate the sdist wheel, we probably want to avoid that.

Reply:
I added a note in the PR description about the SDist.

Reply:
> Can you not recursively copy just the `.gitsubmodule` and exclude the `.git`?

The `.git` folder will be excluded by:

```text
global-exclude .git *~ *.swp
```

at the bottom of `MANIFEST.in`